src/main/java/mate/academy/onlinebookstore/service/cart/ShoppingCartService.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.onlinebookstore.service.cart;
+
+import mate.academy.onlinebookstore.dto.cart.ShoppingCartDto;
+import mate.academy.onlinebookstore.dto.cart.item.AddToCartRequestDto;
+import mate.academy.onlinebookstore.dto.cart.item.UpdateCartItemDto;
+
+public interface ShoppingCartService {
+    ShoppingCartDto addToCart(AddToCartRequestDto requestDto, Long id);
+
+    ShoppingCartDto findById(Long id);
+
+    ShoppingCartDto updateItem(Long userId, Long id, UpdateCartItemDto requestDto);
```

Comment:
In the context of ShoppingCartService it's not obvious which `id` it is, so be more specific.
```suggestion
    ShoppingCartDto updateItem(Long userId, Long cartItemId, UpdateCartItemDto requestDto);
```